docs/docs/cmd/flow/run/run-list.mdx

```diff
@@ -56,10 +61,10 @@ List runs of the specified Microsoft Flow.
 m365 flow run list --environmentName Default-d87a7535-dd31-4437-bfe1-95340acd55c5 --flowName 5923cb07-ce1a-4a5c-ab81-257ce820109a
 ```
 
-List runs of the specified Microsoft Flow with a specific status.
+List runs of the specified Microsoft Flow with a specific status including the trigger information.
 
 ```sh
-m365 flow run list --environmentName Default-d87a7535-dd31-4437-bfe1-95340acd55c5 --flowName 5923cb07-ce1a-4a5c-ab81-257ce820109a --status Running
+m365 flow run list --environmentName Default-d87a7535-dd31-4437-bfe1-95340acd55c5 --flowName 5923cb07-ce1a-4a5c-ab81-257ce820109a --status Running --withTrigger
 ```
```

Comment:
Lets not modify an existing example which is perfectly fine. Instead lets add a new one 👍